voila/handler.py

```diff
@@ -30,7 +31,37 @@
 from .paths import collect_template_paths
 
 
-class VoilaHandler(JupyterHandler):
+class PageConfigMixin:
+    def get_page_config(self):
+        page_config = {
+            "appVersion": __version__,
+            "baseUrl": self.base_url,
+            "terminalsAvailable": False,
+            "fullStaticUrl": url_path_join(self.base_url, "voila/static"),
+            "fullLabextensionsUrl": url_path_join(self.base_url, "voila/labextensions"),
+        }
+
+        mathjax_config = self.settings.get("mathjax_config", "TeX-AMS_HTML-full,Safe")
+        # TODO Remove CDN usage.
```

Comment:
You should be able to depend on this: https://github.com/jupyter-server/jupyter_server_mathjax